README.md

```diff
@@ -14,7 +14,9 @@ for the frontend extension.
 ## Requirements
 
 - JupyterLab >= 4.3.0
-- An IBM Quantum premium account
+- Access to either:
+  - An IBM Quantum premium account
+  - A model with an OpenAI compatible API endpoint
```

Comment:
This is probably too vague, but also hopefully enough for an initial description

Reply:
If you come up with better wording before this is ready to merge feel free to share